src/int/neg.rs

```diff
@@ -0,0 +1,21 @@
+//! [`Int`] negation operation.
+
+use crate::int::Int;
+use crate::WrappingNeg;
+
+impl<const LIMBS: usize> Int<LIMBS> {
```

Comment:
same here, there are two representation of zero so sign should behave correctly i.e.:
-(-0) = +0
-(+0) = -0